README.md

```diff
@@ -211,28 +220,43 @@ BOT_START: Volatility Score >= 6
+### BTC pulse
+BTC pulse is a simple strategy which monitors BTC price action to start new deals or just put the bot to sleep ( no new deals but active deals keep running) based on:-
+If BTC is uptrending new deals are started 
+If BTC is downtrending no new deals are started
 
-BTCPulse hence is determined using the 2 factors :-
+BTC pulse hence is determined using the 2 factors :-
 % price change of BTC in the last 15 minutes or
-Fast and Slow moving EMAs crossses
+Fast (9) and Slow (50) moving EMAs crossses
 
 Please test this strategy on paper before putting real money on it.
 TBMoonWalker or IamtheOnewhoKnocks take no responsibility for losses occurred due to the script/strategy
 
 **Again, please use 3cqsbot only on paper trading. Usage with real funds is at your own risk**
 
+### Fear and Greed Index
+This settings allows you to use the Crypto Fear and Greed index (FGI) to identify the sentiment of the corresponding market phase. The FGI is determined once a day on https://alternative.me/crypto/fear-and-greed-index/
+How to use: when FGI is signaling "greed/very greed" (FGI values usually between 60-100) you may use aggressive DCA settings [fgi_aggressive], e.g. Mars/Banshee/69er covering a price drop of 20-40%.
+
+In phases of fear (FGI values 0-30) over a longer time that may correspond to a beginning or consolidating bear market, the bot can switch to very defensive/conservative DCA settings [fgi_defensive], eg. TA safer, ZachTech BitMan covering a price drop of up to 60%. 
+Get the excel lists from @Snurg at https://discord.com/channels/720875074806349874/829512509798219788/965771867413696532 to get the optimal DCA settings in corresdonding market phases according to your trade funds. 
+
+For sideways market (FGI values 31-60) you can define DCA settings under [fgi_moderate]. 
+
+In each fgi section define the variables fgi_min and fgi_max so that the correct DCA settings are applied. If fgi_min/fgi_max are not set, then values between fgi_min = 0 and fgi_max = 30 are assumed for defensive, FGI values between 31-60 are assumed for moderate and FGI values between 61-100 are assumed for aggressive settings.
+
+If corresponding fgi section is not found, the standard [dcabot] section is used and the FGI values are ignored.
+
+<u>**To know**</u>: No new 3cqsbot is created when using FGI guided DCA settings, instead the new settings are applied to the existent one, so that new deals are started with the new DCA settings.
 
-### Deal Modes
-This section is all about the deal start signals. Tested are the following modes:
+Optionally, the multi pair bot can be renamed according to the prefix, subprefix and suffix given in the corresponding fgi section, e.g. renaming from 3CQSBOT_MULTI_aggressive to 3CQSBOT_MULTI_defensive. To make sure it is always the same bot, you can additionally use the option 'botid' with the same botid number of an already created multi bot in all fgi sections.
 
-- `signal` --> starting the bot after a 3CQS signal
```

Comment:
you have to differentiate - on singlebot, "signal" is deal asap mode on multibot "signal" deal is started with the appropriate start signal (if you have the attribute random_pairs enabled, it will start a random pair from the multibot list)

Reply:
changed